deps/lldb-3.7.1.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/tools/lldb/source/Host/Makefile b/tools/lldb/source/Host/Makefile
+index a8e4260..da90c8c 100644
+--- a/tools/lldb/source/Host/Makefile
++++ b/tools/lldb/source/Host/Makefile
+@@ -14,7 +14,7 @@ include $(LEVEL)/Makefile.config
+ 
+ define DIR_SOURCES
+ SOURCES += $$(addprefix $(1)/,$$(notdir $$(wildcard $$(PROJ_SRC_DIR)/$(1)/*.cpp \
+-					 $$(PROJ_SRC_DIR)/*.cc $$(PROJ_SRC_DIR)/$(1)/*.c)))
++					 $$(PROJ_SRC_DIR)/*.cc $$(PROJ_SRC_DIR)/$(1)/*.c $$(PROJ_SRC_DIR)/$(1)/*.mm)))
+ endef
+ 
+ $(eval $(call DIR_SOURCES,common))
+@@ -22,6 +22,15 @@ $(eval $(call DIR_SOURCES,common))
+ ifeq ($(HOST_OS),Darwin)
+ $(eval $(call DIR_SOURCES,posix))
+ $(eval $(call DIR_SOURCES,macosx))
++CFCPP_SOURCES = \
++          $(addprefix macosx/cfcpp/,$(notdir $(wildcard $(PROJ_SRC_DIR)/macosx/cfcpp/*.cpp)))
++SOURCES += $(CFCPP_SOURCES)
++
++CFCPP_BaseNameSources := $(sort $(basename $(CFCPP_SOURCES)))
++CFCPP_OBJECTS := $(CFCPP_BaseNameSources:%=$(ObjDir)/%.o)
++
++# Make sure the cfcpp output directory exists
++$(CFCPP_OBJECTS): $(ObjDir)/cfcpp/.dir
+ endif
+ 
+ ifeq ($(HOST_OS),Linux)
+@@ -34,8 +43,14 @@ $(eval $(call DIR_SOURCES,posix))
+ $(eval $(call DIR_SOURCES,freebsd))
+ endif
+ 
++ifeq ($(HOST_OS),NetBSD)
++$(eval $(call DIR_SOURCES,posix))
++$(eval $(call DIR_SOURCES,netbsd))
++endif
++
+ ifeq ($(HOST_OS),MingW)
+ $(eval $(call DIR_SOURCES,windows))
++SOURCES += posix/ConnectionFileDescriptorPosix.cpp
```

Comment:
That is from upstream, but I think it was committed directly: https://www.mail-archive.com/lldb-commits@lists.llvm.org/msg03413.html

I'll take it out, although I don't have a windows build right now to test. ... I'm not sure if anyone else actually builds LLVM with MinGW and Makefiles any more.